api/shared.thrift

```diff
@@ -4,6 +4,8 @@
 //   License. See LICENSE.TXT for details.
 // -------------------------------------------------------------------------
 
+const string API_VERSION = '6.0'
```

Comment:
I don't think so. I think this change here was a result of a discussion with the team in person to not have separate API versions for every endpoint.